src/utils/withPageTracker.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import ReactGA from 'react-ga';
+
+export default WrappedComponent =>
+  class WithPageTracker extends Component {
+    currentPage = null;
```

Comment:
I may not understand things here, but even as a class property, there will be one class per page, right?  So this will start out `null`, then be set to the location when the user navigates to this page, and then never be reset again?

For example:
 * user clicks to page A; A.currentPage = "/page/a"; ReactGA.pageview("/page/a");
 * user clicks to page B; B.currentPage = "/page/b"; ReactGA.pageview("/page/b");
 * user clicks to page A; A.currentPage is already "/page/a" so nothing happens